sw/device/tests/aon_timer_smoketest.c

```diff
@@ -44,24 +35,14 @@ static void aon_timer_test_wakeup_timer(dif_aon_timer_t *aon) {
 }
 
 static void aon_timer_test_watchdog_timer(dif_aon_timer_t *aon) {
-  // Make sure that watchdog timer is stopped.
-  CHECK_DIF_OK(dif_aon_timer_watchdog_stop(aon));
-
-  // Make sure the watchdog IRQ is cleared to avoid false positive.
-  CHECK_DIF_OK(
-      dif_aon_timer_irq_acknowledge(aon, kDifAonTimerIrqWdogTimerBark));
-  bool is_pending;
-  CHECK_DIF_OK(dif_aon_timer_irq_is_pending(aon, kDifAonTimerIrqWdogTimerBark,
-                                            &is_pending));
-  CHECK(!is_pending);
-
   // Test the watchdog timer functionality by setting a single cycle "bark"
   // counter. Delay to compensate for AON Timer 200kHz clock (less should
   // suffice, but to be on a cautious side - lets keep it at 100 for now).
-  CHECK_DIF_OK(dif_aon_timer_watchdog_start(aon, 1, 0xffffffff, false, false));
+  aon_timer_testutils_watchdog_config(aon, 1, 0xffffffff);
```

Comment:
You could use the `ULONG_MAX` macro from `sw/device/lib/base/freestanding/limits.h` here instead of `0xffffffff`.

Reply:
Sorry, better yet, use the `UINT32_MAX` in `sw/device/lib/base/freestanding/stdint.h` to be exact.

Reply:
Done